apps/website/src/pages/about/tech/commands.tsx

```diff
@@ -70,6 +71,10 @@ const signature = (command: NamedCommand) => {
 };
 
 const AboutTechPage: NextPage = () => {
+  const copyToClipboard = (command: string) => {
+    navigator.clipboard.writeText(command);
+  };
```

Comment:
Does not need to be in component body.

Reply:
We should abstract this copy method into a util -- the share button also has some copy to clipboard logic, though it uses older DOM methods for doing it. We can probably do both for fallback?

Reply:
Yeah not sure why i did it so complicated back then. I extracted it into a hook.